docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
   # Flask Main Application
   app:
     build: .
-    image: georchestra/foncier-app:18.12
+    image: georchestra/foncier-app:latest
```

Comment:
on a 18.12 branch, that would make sense to keep the tag

Reply:
You're right.

Reply:
the 18.12 tag

Reply:
It should be OK now, no ?

Reply:
not really 
![Capture du 2019-03-12 15-18-53](https://user-images.githubusercontent.com/265319/54207289-2e307e00-44da-11e9-9c77-b2dd79e27c86.png)

Reply:
wait... try reloading the page ? I have 
![Sélection_068](https://user-images.githubusercontent.com/4491521/54209219-ac425400-44dd-11e9-9a79-60ae632881fc.png)
on my side

Reply:
errr ... quantic github ?

Reply:
well, the tag should not be latest but 18.12 on the 18.12 branch

Reply:
Ah, sorry, I thought you wanted the contrary. OK, change it to 18.12

Reply:
I've set 18.12 to worker also. I don't know if it's a good idea or not...

Reply:
Do you want me to change the tags in the Makefile too ? Maybe it would be more consistent

Reply:
Yes please

Reply:
OK, done